File: app/src/main/java/org/mifos/mobilebanking/ui/views/base/OnClickItem.java
package org.mifos.mobilebanking.ui.views.base;

import android.view.View;

/**
 * Created by Rajan Maurya on 23/02/17.
 */

public interface OnClickItem {

    void onItemClick(View childView, int position);

    void onItemLongPress(View childView, int position);
}

File: app/src/main/java/org/mifos/mobile/ui/views/base/OnClickItem.kt
package org.mifos.mobile.ui.views.base

import android.view.View

/**
 * Created by Rajan Maurya on 23/02/17.
 */
interface OnClickItem {
    fun onItemClick(childView: View?, position: Int)
    fun onItemLongPress(childView: View?, position: Int)
}